README.md

```diff
@@ -96,7 +96,25 @@ Assuming you haven’t created any functions yet, the response should be:
 }
 ```
 
-## Step 5: Build, package, and deploy
+## Step 5: Create a bucket for your code
+
+Before you can deploy your code to Stedi Functions, you will need to upload it to [Stedi Buckets](https://www.stedi.com/docs/buckets). You can use any bucket in your account. We recommend you create a separate bucket for deployments.
+
+You can create a bucket using the CLI. Keep in mind that the name needs to be globally unique.
+
+```console
+npx stedi buckets create-bucket --bucket-name <YOUR-BUCKET-NAME-HERE>
```

Comment:
```suggestion
npx stedi buckets create-bucket --bucket-name <YOUR_BUCKET_NAME_HERE>
```

Reply:
You convinced me to make up my mind: https://stedi.slab.com/posts/style-guide-dltapwvo#hzrpx-insert-value-here.

To adhere to this new guideline, I changed the dashes to underscores.

Reply:
awesome!